src/verkle/keyvals/mod.rs

```diff
@@ -0,0 +1,50 @@
+// https://github.com/gballet/verkle-block-sample
+
+use rlp::{Decodable, DecoderError, Rlp};
+use std::convert::TryInto;
+
+#[derive(Debug)]
+pub struct Tuple(Vec<u8>, Vec<u8>);
```

Comment:
name the struct by not what it contains, but what it does. I can already see it's a tuple from it's type, but what this tuple does? why do you need it? or at least what the vecs that it contains?
like `KeyValueTuple` ?